core/trino-server-rpm/src/main/resources/dist/etc/init.d/trino

```diff
@@ -37,6 +37,16 @@ CONFIGURATION=(
     --server-log-file "${SERVER_LOG_FILE:-/var/log/trino/server.log}"
 )
 
+files_limit=131072
+if current_limit=$(ulimit -Hn) && [ "$files_limit" -lt "$current_limit" ]; then
```

Comment:
is it your intention to proceed when `ulimit -Hn` returns non-zero?
yes => document why.
no => move assignment before the `if`.

Reply:
This script doesn't set `-e` and my intention is to avoid comparing `current_limit` if we cannot read it. I'd add a comment like "try to read the current limit and check if it's lower than the default to be set" but this is exactly how you read the condition so I don't think there's much value in a comment like this. Can you suggest a better one?

Reply:
> This script doesn't set `-e`

oh, dear... shouldn't we fix that? (separately)

> my intention is to avoid comparing current_limit if we cannot read it.

then this is the way to do it